README.md

```diff
@@ -215,7 +239,57 @@ class YourClass {
 }
 ```
 
-### Included Collectors
+### Registering Metrics
+
+The best way to register a metric is via a `static final` class variable as is common with loggers.
+
+```java
+static final Counter requests = Counter.build()
+   .name("requests_total").help("Total requests.").labelNames("path").register();
+```
+ 
+Using the default registry with variables that are `static` is ideal since registering a metric with the same name 
+is not allowed and the default registry is also itself static. You can think of registering a metric, more like 
+registering a definition (as in the `TYPE` and `HELP` sections). The metric 'definition' internally holds the samples 
+that are reported and pulled out by Prometheus. Here is an example of registering a metric that has no labels.
+
+```java
+class YourClass {
+  static final Gauge activeTransactions = Gauge.build()
+     .name("my_library_active_transaction")
+     .help("Active transactions.")
+     .register();
+  
+  void processThatCalculates(String key) {
+    activeTransactions.inc();
+    try {
+        // Perform work.    
+    } finally{
+        activeTransactions.dec();
+    }
+  }
+}
+```
+
+To create timeseries with labels, include `labelNames()` with the builder. The `labels()` method looks up or creates 
+the corresponding labelled timeseries. You might also consider storing the labelled timeseries as an instance variable if it is
+appropriate for performance, it is thread safe and can be used multiple times.
```

Comment:
appropriate. It is thread safe and can be used multiple times, which can help performance.